docs/rst/maintainer_guide.rst

```diff
@@ -302,6 +302,23 @@ need to update the lock file.
 
 .. _sec-maintainer_guide-release:
 
+Continuous integration
+----------------------
+
+Eradiate has a continuous integration scheme built in `Github Actions <https://docs.github.com/en/actions>`_ .
+The action is configured in the ``.github/workflows/ci.yml`` file.
+
+As per the documented installation process, Conda environment setup is handled using
+the appropriate Makefile and Mitsuba build configuration is done using the CMake preset.
+No CI-specific build setup operations are required. However to speed up the process, the github action
+uses Mamba instead of Conda to manage the Python environment.
+
+The CI workflow uses caching for the compiled Mitsuba binaries. The cache is identified by the commit hash of the 
+mitsuba submodule and the file hashes of all .cpp and .h files in `src/plugins/src`.
+
+Since the entire pipeline takes more than one hour to complete, it is triggered either manually or when
+a PR is set from `draft` to `ready-for-review`. 
```

Comment:
Does this then mean that the pipeline will run for every commit after marking the PR as ready for review?

Reply:
No, it runs once, when the PR is marked as ready. Now that I think about  it, this doesn't really work for our workflows. I'll check the different triggers again. It is pretty annoying that the comment based trigger only runs on the latest commit of the `main` branch, no matter what you do. See [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_comment-use-issue_comment)